server/verify_test.go

```diff
@@ -36,6 +41,79 @@ import (
 
 var measuredHashes = []crypto.Hash{crypto.SHA1, crypto.SHA256}
 
+func createTpm2EventLog(deviceEnum byte) []byte {
```

Comment:
this should take the GCEConfidentialTechnology enum type.

Reply:
Also, please refactor `TestVerifyAttestationWithSevSnp` so that it uses this function.

Reply:
Done